docs/src/main/sphinx/connector/snowflake.md

```diff
@@ -27,12 +27,18 @@ connector.name=snowflake
 connection-url=jdbc:snowflake://<account>.snowflakecomputing.com
 connection-user=root
 connection-password=secret
+snowflake.private-key=unencrypted-private-key-content
```

Comment:
Does this two needs to be provided when setup?

Reply:
then please remove password from the example configurations, mention it is deprecated in the note section